sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificatesTestBase.cs

```diff
@@ -46,7 +46,7 @@ internal CertificateClient GetClient()
             {
                 Diagnostics =
                 {
-                    IsLoggingContentEnabled = Debugger.IsAttached,
+                    IsLoggingContentEnabled = Debugger.IsAttached || Mode == RecordedTestMode.Live,
```

Comment:
I think `None` is also live. ` != Recorded` might be easier.

Reply:
`None` is a redefinition of `Live` so either should work. I'll try to remember `Recorded` for next time should I need to do this (which I'll remove once I get some useful content).